webextensions/api/declarativeNetRequest.json

```
@@ -148,7 +148,8 @@
               },
```

I reached out to @oliverdunk about Chrome's support of the old property. It appears that while Chrome did not formally mark this property as deprecated, the intent was to replace it. The fact that `nodoc` was added to `MAX_NUMBER_OF_DYNAMIC_AND_SESSION_RULES` in [the same CL](https://source.chromium.org/chromium/chromium/src/+/cbcff7e25340834b9ef92cdb05e20ab4f3a34564) that removed the `nodoc` flags from the new properties is indicative of Chrome's intent to deprecate this property. 

See also WIP Chromium [CL 5637030](https://chromium-review.googlesource.com/c/chromium/src/+/5637030).

```suggestion
                "note": "Deprecated in Chrome 120"
              },
```

---

Resolved.